compiler/infra/Assert.cpp

```diff
@@ -165,4 +168,144 @@ namespace TR
       va_fatal_assertion(file, line, condition, format, ap);
       va_end(ap);
       }
+
+   static bool containsNode(TR::Node *node, TR::Node *target)
```

Comment:
This needs to use a node checklist to avoid an exponential walk

Reply:
Oops, I originally had a checklist here but moved it into another method without thinking when fixing issues with printing commoned nodes. Fixed in e2d71b880deb58289a37bc1534a3015813fe86e2.